README.md

```diff
@@ -104,6 +104,15 @@ The `--fix` option on the command line automatically fixes problems reported by
 | :wrench: | [use-ember-get-and-set](./docs/rules/use-ember-get-and-set.md) | Enforces usage of Ember.get and Ember.set |
 
 
+### Ember Object
```

Comment:
I don't think this section was supposed to move. `yarn update` should generate this file for you.

Reply:
I think after running yarn update only moved this block, i don't remember manually doing it. Anyway restored the position for now